test/integration/AaveV2.t.sol

```diff
@@ -83,51 +83,49 @@ contract AaveV2IntegrationTest is Test {
         vm.label(address(AAVE_V2_PROVIDER), 'AaveV2Provider');
         vm.label(address(pool), 'AaveV2Pool');
         vm.label(address(USDC), 'USDC');
-        vm.label(address(AUSDC_V2), 'aUSDC');
+        vm.label(address(AUSDC_V2_POOL), 'aUSDC');
         vm.label(address(AUSDC_V2_DEBT_VARIABLE), 'variableDebtUSDC');
     }
 
-    function testExecuteAaveV2Borrow(uint256 amountIn) external {
-        vm.assume(amountIn > 1e8);
-        IDebtToken tokenIn = AUSDC_V2_DEBT_VARIABLE;
-        IERC20 tokenOut = IERC20(tokenIn.UNDERLYING_ASSET_ADDRESS());
-        amountIn = bound(amountIn, 1, tokenIn.totalSupply());
-        vm.label(address(tokenOut), 'Token');
+    function testExecuteAaveV2Borrow(uint256 borrowedAmount) external {
+        IDebtToken debtToken = AUSDC_V2_DEBT_VARIABLE;
+        IERC20 borrowedToken = IERC20(debtToken.UNDERLYING_ASSET_ADDRESS());
```

Comment:
IMO, adding an extra line 
```suggestion
        IERC20 borrowedToken = IERC20(debtToken.UNDERLYING_ASSET_ADDRESS());
        IERC20 collateralToken = borrowedToken;
```
should effectively improve the readability.

Reply:
it's been updated. 🙏